src/components/modules/chat/hooks/useUnreadFilter.hook.ts

```
@@ -0,0 +1,55 @@
+        msg => !msg.isRead && msg.sender === "contact"
+      );
+
+      return hasUnreadMessages;
+    });
+  }, [conversations, messages, showUnreadOnly]);
+
+  const unreadCount = useMemo(() => {
+    return conversations.reduce((total, conversation) => {
+      const conversationMessages = messages[conversation.id] || [];
+      const unreadMessages = conversationMessages.filter(
+        msg => !msg.isRead && msg.sender === "contact"
```

ESLint

[error] Replace `msg·=>·!msg.isRead·&&·msg.sender·===·"contact"` with `(msg)·=>·!msg.isRead·&&·msg.sender·===·"contact",`

(prettier/prettier)